src/cms/preview-templates/EnsemblePreview.js

```diff
@@ -11,10 +11,10 @@ const EnsemblesPreview = ({ entry, getAsset }) => {
         image={data.image}
         header={data.header}
         subheading={data.subheading}
-        about={data.about}
         concertAttire={data.concertAttire}
         subheadingTwo={data.subheadingTwo}
         auditionInformation={data.auditionInformation}
+        html={data.html}
```

Comment:
I don't think the page templates work like this. Check https://github.com/netlify-templates/gatsby-starter-netlify-cms/blob/master/src/cms/preview-templates/AboutPagePreview.js for more